.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://di_website:chemical_ali!123!@db:5432/di_website
```

Comment:
Probably should .gitignore the .env!

Reply:
I remember deleting this file and adding it to .gitignore ... hmm

Reply:
https://github.com/devinit/DIwebsite-redesign/commit/5948a5b1b797ccdec52103998cda91161f4605b0#diff-a084b794bc0759e7a6b77810e01874f2

Reply:
Removed unused code from common